:bug: alias addとdeleteが動作しない

### 機能名及び動作

alias add | alias delete

### 再現方法

- add cmd

```bash
$ pummit alias add unko tada
[WARN] Missing alias
```
- delete cmd


```bash
$ pummit alias delete b
panic: runtime error: slice bounds out of range [2:1]

goroutine 1 [running]:
github.com/HidemaruOwO/pummit/pummit/cmd/alias.aliasDelete({0xc00007de50, 0x1, 0x1})
        /home/hidemaru/Code/pummit/pummit/cmd/alias/delete.go:23 +0x306
github.com/HidemaruOwO/pummit/pummit/cmd/alias.AliasDeleteCmd(...)
        /home/hidemaru/Code/pummit/pummit/cmd/alias/delete.go:12
main.main.func3(0xc000197868?)
        /home/hidemaru/Code/pummit/pummit/pummit.go:60 +0x45
github.com/urfave/cli/v2.(*Command).Run(0xc0001c62c0, 0xc0000169c0, {0xc00007ec00, 0x2, 0x2})
        /home/hidemaru/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/command.go:273 +0x9eb
github.com/urfave/cli/v2.(*Command).Run(0xc0001c6000, 0xc000016840, {0xc0000a74d0, 0x3, 0x3})
        /home/hidemaru/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/command.go:266 +0xc4d
github.com/urfave/cli/v2.(*Command).Run(0xc0001c69a0, 0xc000016700, {0xc000016080, 0x4, 0x4})
        /home/hidemaru/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/command.go:266 +0xc4d
github.com/urfave/cli/v2.(*App).RunContext(0xc0001c4000, {0x763298?, 0xc000020068}, {0xc000016080, 0x4, 0x4})
        /home/hidemaru/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/app.go:332 +0x616
github.com/urfave/cli/v2.(*App).Run(...)
        /home/hidemaru/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/app.go:309
main.main()
        /home/hidemaru/Code/pummit/pummit/pummit.go:117 +0x736

$ pummit alias delete b r
[WARN] The alias name to be deleted must be added as the third argument
```

### 詳細な情報

config.jsonにはしっかり絵文字エイリアスが存在している

### その他

_No response_

--- a/pummit/cmd/alias/add.go
+++ b/pummit/cmd/alias/add.go
@@ -12,24 +12,12 @@ import (
 
 var isDebug bool = config.IsDebug()
 
-func AliasAddCmd(args []string) {
-	aliasAdd(args)
+func AliasAddCmd(alias string, prefix string) {
+	aliasAdd(alias, prefix)
 }
 
-func aliasAdd(args []string) {
+func aliasAdd(alias string, prefix string) {
 	aliases := lib.GetAlias()
-
-	if len(args) == 2 {
-		log.Warnf("Missing alias\n")
-		os.Exit(0)
-	}
-	if len(args) == 3 {
-		log.Warnf("Missing emoji prefix\n")
-		os.Exit(0)
-	}
-
-	alias := args[2]
-	prefix := args[3]
 
 	if alias != strings.TrimSpace(alias) || prefix != strings.TrimSpace(prefix) {
 		log.Warnf("Please do not include spaces in alias and emoji prefixes\n")
@@ -76,5 +64,10 @@ func aliasAdd(args []string) {
 	aliases.Alias = append(aliases.Alias, []string{alias, prefix, emoji})
 
 	lib.WriteConfig(aliases)
-	log.Infof("The following has been added to the custom alias\n  %s : %s : %s\n", alias, prefix, emoji)
+	log.Infof(
+		"The following has been added to the custom alias\n  %s : %s : %s\n",
+		alias,
+		prefix,
+		emoji,
+	)
 }

--- a/pummit/cmd/alias/delete.go
+++ b/pummit/cmd/alias/delete.go
@@ -1,42 +1,35 @@
 package alias_cmd
 
 import (
-	"os"
 	"sync"
 
 	"github.com/HidemaruOwO/nuts/log"
 	"github.com/HidemaruOwO/pummit/pummit/lib"
 )
 
-func AliasDeleteCmd(args []string) {
-	aliasDelete(args)
+func AliasDeleteCmd(targets []string) {
+	aliasDelete(targets)
 }
 
-func aliasDelete(args []string) {
+func aliasDelete(targets []string) {
 	alias := lib.GetAlias()
+	slice := alias.Alias
 
-	if len(args) == 2 {
-		log.Warnf("The alias name to be deleted must be added as the third argument\n")
-		os.Exit(0)
-	} else {
-		slice := alias.Alias
-		targets := args[2:]
-		log.Debugf(isDebug, "Removing %v\n", targets)
+	log.Debugf(isDebug, "Removing %v\n", targets)
 
-		var wg sync.WaitGroup
-		wg.Add(len(targets))
-		for _, t := range targets {
-			go func(target string) {
-				defer wg.Done()
-				slice = removeSlice(slice, target)
-				log.Infof("Removed %s alias\n", target)
-			}(t)
-		}
-		wg.Wait()
-
-		alias.Alias = slice
-		lib.WriteConfig(alias)
+	var wg sync.WaitGroup
+	wg.Add(len(targets))
+	for _, t := range targets {
+		go func(target string) {
+			defer wg.Done()
+			slice = removeSlice(slice, target)
+			log.Infof("Removed %s alias\n", target)
+		}(t)
 	}
+	wg.Wait()
+
+	alias.Alias = slice
+	lib.WriteConfig(alias)
 }
 
 func removeSlice(slice [][]string, target string) [][]string {

--- a/pummit/pummit.go
+++ b/pummit/pummit.go
@@ -29,7 +29,12 @@ func main() {
 	app := &cli.App{
 		EnableBashCompletion: true,
 		Action: func(ctx *cli.Context) error {
-			cmd.RootCmd(ctx.Args().Get(0), ctx.Args().Get(1))
+			if ctx.Args().Present() {
+				cmd.RootCmd(ctx.Args().Get(0), ctx.Args().Get(1))
+			} else {
+				return fmt.Errorf("Error: %s",
+					"Arguments must have both a prefix and an emoji")
+			}
 
 			return nil
 		},
@@ -49,7 +54,14 @@ func main() {
 					Name:  "add",
 					Usage: "Add an alias for the git character.",
 					Action: func(ctx *cli.Context) error {
-						alias_cmd.AliasAddCmd(ctx.Args().Slice())
+						if ctx.Args().Present() {
+							alias_cmd.AliasAddCmd(ctx.Args().Get(0), ctx.Args().Get(1))
+						} else {
+							return fmt.Errorf(
+								"Error: %s",
+								"Arguments must have both a prefix and an emoji",
+							)
+						}
 						return nil
 					},
 				},
@@ -57,7 +69,13 @@ func main() {
 					Name:  "delete",
 					Usage: "Remove aliases for git characters.",
 					Action: func(ctx *cli.Context) error {
-						alias_cmd.AliasDeleteCmd(ctx.Args().Slice())
+						if ctx.Args().Present() {
+							alias_cmd.AliasDeleteCmd(ctx.Args().Slice())
+						} else {
+							return fmt.Errorf("Error: %s",
+								"delete requires one or more pairs of prefix and emoji in arguments")
+						}
+
 						return nil
 					},
 				},

